From 4793d53acac337890170e0f475747f7b681bf42b Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao G <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Fri, 16 Feb 2018 10:20:59 +0530
Subject: [PATCH 0280/1852] dt-bindings: display: xlnx: Added vpss bridge
 phandle as optional property

commit f59cf64e0260874c7787e358234d22dcd88b101d from
https://github.com/Xilinx/linux-xlnx.git

This patch adds the phandle entry of VPSS bridge.

Signed-off-by: Venkateshwar Rao G <venkateshwar.rao.gannavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt
index 5b56ac248575..0079c4c6391c 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt
@@ -38,6 +38,10 @@ Required simple_panel properties:
    For available panel compatible strings, please refer to bindings in
    Documentation/devicetree/bindings/display/panel/
 
+Optional properties:
+ - xlnx,vpss: vpss phandle
+   This handle is required only when VPSS is connected to DSI as bridge.
+
 Example:
 
 #include <dt-bindings/drm/mipi-dsi.h>
@@ -48,6 +52,7 @@ Example:
 		xlnx,dsi-data-type = <MIPI_DSI_FMT_RGB888>;
 		#address-cells = <1>;
 		#size-cells = <0>;
+		xlnx,vpss = <&v_proc_ss_0>;
 		encoder_dsi_port: port@0 {
 			reg = <0>;
 			dsi_encoder: endpoint {
-- 
2.31.1

